src/connection_string.ts

```diff
@@ -77,14 +79,12 @@ export function resolveSRVRecord(options: MongoOptions, callback: Callback<HostA
     if (err) return callback(err);
 
     if (addresses.length === 0) {
-      return callback(new MongoParseError('No addresses found at host'));
+      return callback(new MongoURIError('No addresses found at host'));
```

Comment:
So, the URI error now extends a runtime error, but this is misleading because its description still says it's the user's fault; moreover, if we are intending to introduce a MongoConnectionStringError, having both of these errors is going to be confusing; I think the best thing to do might be to remove the MongoURIError entirely, and just replace the errors here with MongoAPIError or MongoRuntimeError as appropriate (note, in L289 and L404, it is an API error)

Reply:
Oops, complete oversight on my part! Replacing this with MongoAPIError and removing the class definition. So sorry!!